aim/sdk/adapters/hugging_face.py

```diff
@@ -108,15 +114,38 @@ def on_log(self, args, state, control,
                     )
                 continue
 
-            self._run.track(log_value,
-                            name=log_name, context=context,
-                            step=state.global_step, epoch=state.epoch)
+            self._run.track(
+                log_value,
+                name=log_name,
+                context=context,
+                step=state.global_step,
+                epoch=state.epoch,
+            )
 
     def close(self):
         if self._run:
             self._run.close()
             del self._run
             self._run = None
 
+    @staticmethod
+    def find_most_common_substring(names: List[str]) -> Dict[str, int]:
+        substring_counts: Dict[str, int] = {}
```

Comment:
@tmynn I'm not sure I fully understand the code below.
As we're getting only the first key of the dict, does it make sense to fill in the rest of matches? 🤔

Reply:
@alberttorosyan good point! Need to take the key with max value